card/mod/01_core/set/all/permissions.rb

```diff
@@ -72,7 +72,7 @@ def deny_because why
 
 def permitted? action
 
-  if !Wagn.config.read_only
+  if !Cardio.config.read_only
```

Comment:
Sure, that should be fine.  If there are places where Card.config ... doesn't work (and Card.paths and Card.root) then Cardio.x will still work.